NotificationApi/NotificationApi/Controllers/ParticipantEmailNotificationsController.cs

```diff
@@ -148,19 +146,15 @@ public async Task<IActionResult> SendParticipantWelcomeEmailAsync(NewUserWelcome
         public async Task<IActionResult> SendParticipantSingleDayHearingConfirmationForExistingUserEmailAsync(
             ExistingUserSingleDayHearingConfirmationRequest request)
         {
-            var useNewTemplates = _featureToggles.UsePostMay2023Template();
             var notificationType = request.RoleName switch
             {
-                RoleNames.Individual when !useNewTemplates => NotificationType.HearingConfirmationLip,
-                RoleNames.Individual when useNewTemplates => NotificationType.ExistingUserLipConfirmation,
-                RoleNames.Representative when !useNewTemplates=> NotificationType.HearingConfirmationRepresentative,
-                RoleNames.Representative when useNewTemplates => NotificationType.ExistingUserRepresentativeConfirmation,
+                RoleNames.Individual => NotificationType.ExistingUserLipConfirmation,
+                RoleNames.Representative => NotificationType.ExistingUserRepresentativeConfirmation,
                 RoleNames.JudicialOfficeHolder when !request.Username.IsJudiciaryUsername() => NotificationType
                     .HearingConfirmationJoh,
                 RoleNames.JudicialOfficeHolder when request.Username.IsJudiciaryUsername() => NotificationType
                     .HearingConfirmationEJudJoh,
-                RoleNames.Judge when !request.Username.IsJudiciaryUsername() => NotificationType.HearingConfirmationJudge,
-                RoleNames.Judge when request.Username.IsJudiciaryUsername() => NotificationType.HearingConfirmationEJudJudge,
+                RoleNames.Judge => NotificationType.HearingConfirmationEJudJudge,
```

Comment:
I don't think `NotificationType.HearingConfirmationJudge` is behind a feature toggle but rather `request.Username.IsJudiciaryUsername()`

Reply:
reinstated